src/Glpi/Console/Migration/FormCreatorPluginToCoreCommand.php

```diff
@@ -0,0 +1,179 @@
+<?php
+
+/**
+ * ---------------------------------------------------------------------
+ *
+ * GLPI - Gestionnaire Libre de Parc Informatique
+ *
+ * http://glpi-project.org
+ *
+ * @copyright 2015-2025 Teclib' and contributors.
+ * @copyright 2003-2014 by the INDEPNET Development Team.
+ * @licence   https://www.gnu.org/licenses/gpl-3.0.html
+ *
+ * ---------------------------------------------------------------------
+ *
+ * LICENSE
+ *
+ * This file is part of GLPI.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ *
+ * ---------------------------------------------------------------------
+ */
+
+namespace Glpi\Console\Migration;
+
+use Glpi\Console\AbstractCommand;
+use Glpi\Form\Migration\MigrationManager;
+use Glpi\Helpdesk\DefaultDataManager;
+use Symfony\Component\Console\Exception\RuntimeException;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+
+class FormCreatorPluginToCoreCommand extends AbstractCommand
+{
+    /**
+     * Error code returned if plugin version or plugin data is invalid.
+     *
+     * @var integer
+     */
+    public const ERROR_PLUGIN_VERSION_OR_DATA_INVALID = 1;
+
+    /**
+     * Error code returned if import failed.
+     *
+     * @var integer
+     */
+    public const ERROR_PLUGIN_IMPORT_FAILED = 1;
```

Comment:
Not sure these error code bring a lot as they have the same value as the default `Command::FAILURE` return.

It would maybe be more clear to use them as error message instead ? Right now the errors are not explained to the user and can't be traced back using the code either.